analyzer-comments/elixir/top-secret/function_capture.md

```diff
@@ -0,0 +1,14 @@
+# function capture
+
+When passing an existing function as an argument, make sure to capture it instead of creating a new anonymous function.
+
+```elixir
+# preferred:
+# capture the existing function
+&decode_secret_message_part/2
+
+# not preferred: 
+# creating new anonymous functions
+&decode_secret_message_part(&1, &2)
+fn ast, acc -> decode_secret_message_part(ast, acc) end
```

Comment:
is it clear here that those are two examples of different ways of creating anonymous functions?

Reply:
The comment makes it pretty clear, yes. If you have a doubt you could go make it extra-clear: 
```
# two equivalent examples of creating new anonymous functions
```

Reply:
ok, great :) I prefer it with fewer words so I'll leave it like it is